examples/set-namespace/helloworld/README.md

```diff
@@ -0,0 +1,76 @@
+Set Namespace
+==================================================
+
+Function `set-namespace` sets namespaces in KRM objects.
+
+## Get Image
+
+If you want to use the cutting-edge version of this function, you need to clone
+the entire repo locally and then run the following to build the image:
+
+<!-- @buildImage @testMaster -->
+```
+TAG=master
+IMAGENAME=gcr.io/kpt-functions-trusted/set-namespace:$TAG
+docker build -t $IMAGENAME ../../../functions/go/set-namespace/
+```
+
+If you want to use the latest released version:
+
+<!-- @selectImage @testStable -->
+```
+TAG=v0.1
```

Comment:
If we hardcode this TAG here, it will not be the latest version. Maybe a TODO to proper handle this TAG after we finalize the versioning.